### ISSUE TITLE
iOS TensorflowSharp build failed with: Undefined symbols for architecture arm64:   "_cblas_sgemm"

### ISSUE THREAD

**Issue by vanIvan** [Unity-Technologies/ml-agents#1397]
While building for iOS, following [documentation](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Using-TensorFlow-Sharp-in-Unity.md) this error occurs during linking phase:
```
Undefined symbols for architecture arm64:
  "_cblas_sgemm", referenced from:
       tensorflow::Conv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::Im2ColConvFunctor<float, float, float, FastGemmFunctor<float, float, float> > >::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_using_gemm.o)
      tensorflow::FusedResizeConv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::FusedResizeAndPadConvFunctor<float, float, float, FastGemmFunctor<float, float, float>, (tensorflow::(anonymous namespace)::SamplingMode)0>, true>::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_fused.o)
      tensorflow::FusedResizeConv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::FusedResizeAndPadConvFunctor<float, float, float, FastGemmFunctor<float, float, float>, (tensorflow::(anonymous namespace)::SamplingMode)1>, false>::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_fused.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

It might be solved by adding [Accelerate framework](https://developer.apple.com/documentation/accelerate) shipped with Xcode following this [guide](http://docs.onemobilesdk.aol.com/ios-ad-sdk/adding-frameworks-xcode.html) or going through next steps: in **Project** > **Build Phases** > **Link Binary With Libraries** > **+** (add from suggested list)

**Comment by xiaomaogy** [comment]
Hi @vanIvan, thanks for your note. We used to ask people to add Accelerate framework specifically, but in v0.5 we realized that if you have checked Platform Settings -> Rarely Used Frameworks -> Accelerate in Plugins/iOS/TensorFlowSharp.dll and build it for iOS, the Accelerate framework will be linked by default, so we removed that step starting in v0.5. 

Are you sure you are using the latest version of TensorFlowSharp plugin? If you used the version for v0.4, it might not be linked by default.
